samples/golf/CMakeLists.txt

```diff
@@ -43,9 +43,6 @@ if(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
   set(CMAKE_CXX_FLAGS  "${CMAKE_CXX_FLAGS} -Wno-potentially-evaluated-expression")
 endif()
 
-SET(CMAKE_CXX_FLAGS_DEBUG "-g -DCRO_DEBUG_")
```

Comment:
Similar to the change in the crogine CMakeLists, the `-g` and `-O3` are not cross platform and should be the defaults from cmake anyway, so removed and replaced the `CRO_DEBUG_` definition with the modern alternative below